source/_posts/en/tutorial/ssr.md

```diff
@@ -1,40 +1,39 @@
 ---
-title: 服务端渲染
+title: Server Side Rendering
 categories:
 - tutorial
 ---
 
 
 
-San 的服务端渲染支持是基于 [组件反解](../reverse/) 的：
+San's server-side rendering is based on [component inverse](../reverse/).
 
-- 服务端输出的 HTML 中带有对视图无影响，能帮助组件了解数据与视图结构的标记片段
-- 浏览器端，组件初始化时从标记片段理解组件结构，在后续用户操作时组件能正确响应，发挥作用
+- The HTML output from the server has a markup fragment that has no effect on the view and helps the component understand the structure of the data and view.
+- On the browser side, the component structure is understood from the markup fragment when the component is initialized, and the component can respond correctly when the user operates.
 
-`提示`：由于组件运行环境需要考虑浏览器各版本和NodeJS，示例代码为保证简单无需transform，全部采用ES5编写。
+`tips`: Since the component runtime environment needs to consider browser versions and NodeJS, the sample code is guaranteed to be simple and no transform is required, all written in ES5.
 
-是否需要SSR
+Do you need SSR?
 ----
 
-服务端渲染，视图 HTML 直出有一些显而易见的好处：
+Server-side rendering, view HTML straight out has some obvious benefits:
```

Comment:
We can gain some obvious benefits from server-side rendering which outputs HTML directly